#CLoud Info
Header header 

int64 imuAvailable
int64 odomAvailable

float32 imuRollInit
float32 imuPitchInit
float32 imuYawInit

float32 initialGuessX
float32 initialGuessY
float32 initialGuessZ
float32 initialGuessRoll
float32 initialGuessPitch
float32 initialGuessYaw

sensor_msgs/LaserScan scan